src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts

```diff
@@ -196,6 +229,26 @@ class indexErrors extends viewModelBase {
         });
     }
 
+    private extractActionNamesAndCount(indexErrors: Raven.Client.Documents.Indexes.IndexErrors[]): Array<indexActionAndCount> {
+        const mappedItems = _.flatMap(indexErrors,
+            value => {
+                return value.Errors.map(x => ({
+                    actionName: x.Action,
+                    count: 1
+                } as indexActionAndCount));
+            });
+
+        return mappedItems.reduce((result: indexActionAndCount[], next: indexActionAndCount) => {
```

Comment:
Can we sort the result by actionName?

Reply:
Sorting is also needed for the index name

Reply:
done